UQ-eLIPSE/crucible-components
# Feature/vethub 276

SRS only apply for Tutor mode and can be enable or disable
- to test: 'yarn test'
- to verify: visit testsso Zone
- 
![image](https://github.com/user-attachments/assets/f50efc1f-9108-4f91-bff1-dcdc492fff3c)
